libs/contact/src/driver/hubspot/public_api.ts

```diff
@@ -0,0 +1,3 @@
+export { DaffContactHubspotTransformer } from './transformers/contact.transformer';
+export { DaffContactHubspotService } from './contact.service';
```

Comment:
I don't think we want to export either of these concrete services, because we don't want the app dev ever to import them or try something weird like:
```
{provide: DaffContactHubspotService, useClass: SomeOtherCustomService}
```
We want them to implement the corresponding interfaces instead.